etc/schema.json

```diff
@@ -2428,34 +2428,88 @@
             "additionalProperties": false,
             "properties": {
                 "cname": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "The client PrincipalName",
+                    "suricata": {
+                        "keywords": [
+                            "krb5.cname"
+                        ]
+                    }
                 },
                 "encryption": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "Encryption used (only in AS-REP and TGS-REP)",
+                    "suricata": {
+                        "keywords": []
```

Comment:
Have we applied any significance to an empty keywords list? We use `false` for no keyword mapping where there never should be one.

Reply:
Yeah, thats what we have already.  Here is where I might recommend a "$comment" as a note to readers of the schema..

```
"suricata": {
    "$comment": "additional metadata, no keyword",
    "keywords": false
}
```

Reply:
I think it's a bug/oversight in the parity script that it treats an empty list as "false".  Both are falsy in Python.

Reply:
Adding comments that for these cases, we should indeed add keywords